### PR TITLE
Added Contact form

### PR DESCRIPTION
**Pull request by daveblaq**
### Added Contact form
**On this webpage I collaborated with my teammates on the pair programming section and we:**

- Added Contact section and title
- Added Contact form

### CODE REVIEW

**Review by zhadier — CHANGES_REQUESTED** [review]
Made a review by mistake :,) please bear with.

File: index.html [hunk 1]
```diff
@@ -242,6 +237,30 @@ <h4>Skills</h4>
...
+    </section>
+    <section class="contact-me">
+        <div class="contact-title">
+            <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3> 
+        </div>
+        <div class="contact-form">
+            <form action="https://formspree.io/f/meqnbegz" method="post">
+                <input type="text" id="name" name="name" class="input-name" placeholder="Full Name" required maxlength="30">
+                <input type="email" id="email" name="email" class="input-email" placeholder="Email address" required>
+                <textarea name="message" id="message" required class="input-text-area" placeholder="Enter text here"></textarea>
```
Comment (zhadier) [inline index.html]: - [x] Great job on the form validations ✔️. One tiny last validation you have to do is to make sure the message input has a max length of 500. Kindly add this validation aswell 🌤️

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @daveblaq  👋 

Good job so far! 🌤️ 
There are some issues that you still need to work on to go to the next project but you are almost there! 🍓 



## Highlights
✔️ No linter errors
✔️ Github flow was set up correctly
✔️ Professional pull request title and description
✔️ You have done well matching your design to the original Figma design
✔️ You have used flexbox well for aligning the elements
✔️ Uses the font and colors (background and font) defined for the template

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (daveblaq) [reply index.html]: This has been done. Thank you

**Review by blessedjasonmwanza — APPROVED** [review]
# 👀  STATUS: APPROVED  💯 
 ## 🤓  Hi @daveblaq ,
> Great work on making the changes requested by the previous reviewer :white_check_mark:
> 
> Your project is complete! There is nothing else to say other than... Great job :clap: Congratulations! :tada:
> 
> Now you can merge this branch and move on :1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:
> 
> 
> 
> Cheers and Happy coding! :clap: :clap: :clap:
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**